CHANGELOG.md

```diff
@@ -4,7 +4,28 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## 1.7.0 (unreleased)
+## 1.7.0 (2020-04-16)
+
+This release adds major new features and bugfixes since the 1.6.1 release.
+We deem it moderate priority for upgrading.
+
+This release adds the long-awaited support for PostgreSQL 12 to timescaledb.
+
+This release also adds a new mode for continuous aggregates that we name
+real-time aggregates. Unlike the original this new mode will
+combine materialized data with new data received after the last
+refresh has happened. This new mode will be the default behaviour
+for newly created continuous aggregates.
+
+To upgrade existing continuous aggregates to the new behaviour
+the following command needs to be run for all continuous aggregates
```

Comment:
Propose clarifying that this is a query behavior:

```suggestion
This release also adds a new default behavior when querying continuous aggregates that we call
real-time aggregation. A query on a continuous aggregate will now combine materialized data with recent data that has yet to be materialized. Note that only newly created continuous aggregates will have this real-time query behavior, although it can be enabled on existing continuous aggregates with a configuration setting as follows:
```